packages/components/src/InputField/Checkbox.purs

```diff
@@ -0,0 +1,81 @@
+module KSF.InputField.Checkbox where
```

Comment:
Just going to call this module checkbox. Trying to make radio buttons and checkboxes behave nicely and logically within the same module is not feasible imo. They are just too different in their behaviour.